docker/package/tezos_voting_wizard.py

```diff
@@ -125,6 +119,44 @@ def get_proposal_period_hash(hashes):
 )
 
 
+def get_node_rpc_addr_query(default=None):
+    return Step(
+        id="node_rpc_addr",
+        prompt="Provide the node's RPC address.",
+        help="The node's RPC address will be used by tezos-client to vote. If you have baking set up\n"
+        "through systemd services, the address is usually 'http://localhost:8732' by default.",
+        default=default,
+        validator=Validator(
+            [
+                required_field_validator,
+                reachable_url_validator("chains/main/blocks/head/"),
```

Comment:
Minor annoyance: it seems that this step requires the URL to be given with a `/` at the end or it won't work.

Moreover the error message will say:
```
Validation error: chains/main/blocks/head/ is unreachable. Please input a valid URL.
```
that doesn't make what the problem is very clear.

Reply:
If you input the full URL including the scheme, it works as intended 🤔 I've made a fix with regards to correct URL joining, but it will only help with the error reporting, as `urllib.request.urlopen` doesn't accept links without a scheme anyway, nor do I think we should make it.